openerp/addons/openupgrade_records/model/install_all_wizard.py

```diff
@@ -1,28 +1,6 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    This module Copyright (C) 2012-2014 OpenUpgrade community
-#    https://launchpad.net/~openupgrade-committers
-#
-#    Contributors:
-#    Therp BV <http://therp.nl>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
+# © 2012-2017 Therp BV <http://therp.nl>
```

Comment:
Agreed with the new short headers, but please add copyright instead of replacing it.